### DISCUSSION TITLE
Explanation logging in to Youtube music

### DISCUSSION THREAD

**Discussion by siggi1984** [z-huang/music#404] [General]
Cool would be a small explanation of the new function: Logging in to Youtube Music.
 Are there then personalized music suggestions?

Can I access the playlists created at YTM?

Are the playlists created with InnerTune automatically synchronized with YTM?

And so on....?

**Comment by NeeRaj-2401** [comment]
Here is [Explanation](https://github.com/z-huang/InnerTune/issues/88#issuecomment-1302152845), Also u can track everything related to login [here](https://github.com/z-huang/InnerTune/issues/88)

**Comment by z-huang** [comment]
Let's move the conversation to #88.
